fleet/lib/htcondor/fleet.yaml

```diff
@@ -0,0 +1,30 @@
+---
+defaultNamespace: htcondor
+labels:
+  bundle: &name htcondor
+dependsOn:
```

Comment:
I haven't gotten around to doing IT-5248 yet but the idea to trim down the deps to only crds or resources like secrets, which if that don't exist at the time of deployment the helm chart will create it from scratch.

I believe the only CRD being depended on here is `bundle: external-secrets` (no `-conf`).

Reply:
this was modified 👍🏻